astroquery/mpc/core.py

```diff
@@ -1088,24 +1094,27 @@ def _parse_result(self, result, **kwargs):
                 elif 's=s' in result.request.body:  # sky Motion
                     names += ('dRA cos(Dec)', 'dDec')
                     units += ('arcsec/h', 'arcsec/h')
-                col_starts += (73, 81)
-                col_ends += (80, 89)
+                # Correct start and end for Motion columns if the results have 3 digit precision.
+                col_starts += (73, 82)
+                col_ends += (81, 91)
 
                 if 'Moon' in columns:
                     # table includes Alt, Az, Sun and Moon geometry
                     names += ('Azimuth', 'Altitude', 'Sun altitude', 'Moon phase',
                               'Moon distance', 'Moon altitude')
+                    # Modified column start and end as motion column can have 3 digit precision.
                     col_starts += tuple((col_ends[-1] + offset for offset in
-                                         (2, 9, 14, 20, 27, 33)))
+                                        (1, 8, 13, 19, 26, 32)))
                     col_ends += tuple((col_ends[-1] + offset for offset in
-                                       (8, 13, 19, 26, 32, 37)))
+                                      (7, 12, 18, 25, 31, 36)))
                     units += ('deg', 'deg', 'deg', None, 'deg', 'deg')
                 if 'Uncertainty' in columns:
                     names += ('Uncertainty 3sig', 'Unc. P.A.')
+                    # Modified column start and end as motion column can have 3 digit precision.
```

Comment:
I don't think this comment is needed, as the pull request documents the change.

Reply:
I have removed the comment line